graviti/client/draft.py

```diff
@@ -83,7 +143,34 @@ def update_draft(  # pylint: disable=unused-argument
         title: The draft title.
         description: The draft description.
 
-    Return:
-        The response of OpenAPI.
+    Examples:
+        Update the title or description of the draft:
+        >>> update_draft(
+        ...     "https://gas.graviti.com/",
+        ...     "ACCESSKEY-********",
+        ...     "2bc95d506db2401b898067f1045d7f68",
+        ...     2,
+        ...     title="draft-3"
+        ... )
+
+        Close the draft:
+        >>> update_draft(
+        ...     "https://gas.graviti.com/",
+        ...     "ACCESSKEY-********",
+        ...     "2bc95d506db2401b898067f1045d7f68",
+        ...     2,
+        ...     status="CLOSED"
+        ... )
 
     """
+    url = urljoin(url, f"{PARTIAL_URL}/datasets/{dataset_id}/drafts/{draft_number}")
+    patch_data: Dict[str, Any] = {"draftNumber": draft_number}
+
+    if status:
+        patch_data["status"] = status
+    if title is not None:
```

Comment:
Can `title` be an empty string?

Reply:
no, but we need to pass in the `title`, then tell the user this restriction by error message.